yoti-sdk-api/src/main/java/com/yoti/api/client/DigitalIdentityClient.java

```diff
@@ -28,11 +31,21 @@ public class DigitalIdentityClient {
         this.identityService = identityService;
     }
 
-    public Object createShareSession() {
+    /**
+     * Create a sharing session to initiate a sharing process based on a policy
+     *
+     * @param request
+     *              Details of the request like policy, extensions and push notification for the application
+     * @return an {@link ShareSession}
+     *              Id, status and expiry of the newly created Share Session
+     * @throws DigitalIdentityException
+     *              Aggregate exception signalling issues during the call
+     */
```

Comment:
This java doc doesn't say much beyond what the method name already says
- `Create a sharing session` -> `createShareSession`
- All params descriptions are essentially saying what could be determined by looking at the API of either `ShareSession` or `ShareSessionRequest`
- `Aggregate exception signalling issues during the call` that's basically a description of _any_ exception, by definition it means "there were issues" that prevented normal execution

I don't have a fancy suggestion for the javadoc here, but usually if I cannot find words to inform beyond what the code already explains then might as well not add any javadoc.

Reply:
I was just following what the current "(non)standard" is in place. IMO we could simply not provide any java docs